api_pagopa.yaml

```diff
@@ -123,6 +123,10 @@ definitions:
         exclusiveMaximum: true
       detail:
         $ref: "#/definitions/PaymentFault"
+      detail_v2:
+        type: string
```

Comment:
is it possible to keep the type of this field as `enum string` instead of a `string`?
Since, in most case, the code is auto-generated starting from the _spec_ it could be very useful handling enums instead of strings

Reply:
Yes we could done it in to new PR , but shoud be careful to [new enum type](https://github.com/pagopa/pagopa-nodo4-nodo-dei-pagamenti/blob/0c921994609031a39149952204cb8192a85a1971/common/src/main/scala/eu/sia/pagopa/common/util/DigitPaErrorCodes.scala#L4) that should contains several values

Reply:
Thank you. I prefer a huge enum instead of a typo we could (surely) write in the error handling :hurtrealbad:

Reply:
coming soon https://pagopa.atlassian.net/browse/PAG-756 🚀  @infantesimone